addon_service/external_storage_service/models.py

```diff
@@ -19,15 +19,18 @@ class ExternalStorageService(AddonsServiceBaseModel):
     int_addon_imp = models.IntegerField(
         null=False,
         validators=[validate_storage_imp_number],
+        verbose_name="Addon implementation",
     )
     int_credentials_format = models.IntegerField(
         null=False,
         validators=[validate_credentials_format],
+        verbose_name="Credentials format",
     )
     int_service_type = models.IntegerField(
         null=False,
         default=ServiceTypes.PUBLIC.value,
         validators=[validate_service_type],
+        verbose_name="Service type",
```

Comment:
Let's explicitly add `HYBRID = PUBLIC | HOSTED` to the ServiceTypes Flag so that it's selectable in Admin.

Reply:
turns out iterating thru an `enum.Flags` omits the aliases -- had to update to use `__members__`